templates/confidential-computing/index.html

```diff
@@ -42,12 +122,12 @@ <h2>How confidential VMs work</h2>
         </p>
         <div class="p-image__background--3-2 u-align--center u-vertically-center">
           {{ image (
-            url="https://assets.ubuntu.com/v1/abd7bccf-Hypervisor%20diagram%203.png",
-            alt="Hypervisor diagram",
-            width="195",
-            height="282",
-            hi_def=True,
-            loading="lazy" ) | safe
+          url="https://assets.ubuntu.com/v1/abd7bccf-Hypervisor%20diagram%203.png",
+          alt="Hypervisor diagram",
```

Comment:
Can you write a more descriptive alt text here please, the template I use for charts is '<chart type> of <type of data> where <reason for including chart>'

Reply:
@Sophie-32 Can you please suggest a text for this? And also for the new diagram please

Reply:
I'm trying to get in touch with Felicia to learn more about the graphics since I don't understand these graphics well. In the meantime, since they appear to be described by their intro text. I suggest: First graphic - 'Graphic showing how Ubuntu supports Intel's confidential computing technology.' Second graphic - 'Graphic showing that confidential virtual machines create a boundary around their software and keep other softwares out.'